nncf/torch/dynamic_graph/patch_pytorch.py

```diff
@@ -289,6 +290,11 @@ def patch_torch_jit():
 
     import torch
 
+    try:
+        import torchvision  # noqa
+    except:
+        pass
```

Comment:
Before, there were recorded some issues when `torchvision` was imported before `nncf.torch`  (ticket 105042). The ticket (109653) to improve this is also still open. 

If I remember correctly, in that particular case to make it work we had to ask OTX team to move `nncf.torch` import higher in a hierarchy than import of `torchvision`.

There is a reproduction test attached to the latter ticket. There, `torchvision` is imported before `nncf.torch` to make the issue reproduce. @anzr299 could you please check that if we remove `torchvision` import from the reproducer and apply your changes, the the reproducer won't fail?

Also, some test which fails without these changes would be appreciated. Of course, if there isn't one already.

Reply:
Yes, you are right. I had noticed some of the example tests fail for the current PR's approach of importing torchvision. After noticing this issue, I am amidst the process of fixing it. 
The tests which fail due to torchvision import **after** NNCF is `tests/torch/fx/test_bias_correction`.

Reply:
> Yes, you are right. I had noticed some of the example tests fail for the current PR's approach of importing torchvision. After noticing this issue, I am amidst the process of fixing it.
The tests which fail due to torchvision import after NNCF is tests/torch/fx/test_bias_correction.

I see, thanks!

> @anzr299 could you please check that if we remove torchvision import from the reproducer and apply your changes, the the reproducer won't fail?

Did you run the mentioned reproducer with the described changes?

Reply:
Yes, I had rerun the test and it works perfectly fine.
This is the model which requires positioning of torchvision: `examples/post_training_quantization/torch/ssd300_vgg16/main.py`

Reply:
Ok, thanks!